src/test/java/nl/hsac/fitnesse/util/iban/ATIbanTest.java

```diff
@@ -0,0 +1,65 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests AT Iban generator.
+ */
+public class ATIbanTest {
```

Comment:
Basically all country specific tests are the same:
- Check if the result starts with the country code
- Check if the length is correct
- Test without bankcode given
- Test with bankcode given
- Test with some random bankcode

So yes, some superclass should work.